python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -566,6 +576,7 @@ async def do_offset(self, data):
         See note for do_move regarding checking the target position.
         """
         self.assert_enabled()
+        self.move_event.set()
```

Comment:
Should we set the event after knowing the low-level controller accepts this command? Under self._move()?